packages/devtools_app_shared/lib/src/ui/theme/theme.dart

```diff
@@ -332,6 +332,8 @@ double get mediumProgressSize => scaleByFontFactor(24.0);
 const defaultTabBarViewPhysics = NeverScrollableScrollPhysics();
 
 // Font size constants:
+const unscaledExtraLargeFontSize = 16.0;
```

Comment:
nit: if we are only using this in the property editor, can we just provide this constant there instead of adding something to devtools_app_shared?